src/Service/Services/MetadataProviders/CosmosSqlMetadataProvider.cs

```diff
@@ -235,5 +235,10 @@ public string GetDefaultSchemaName()
         {
             return string.Empty;
         }
+
+        public bool IsDevelopmentMode()
+        {
+            return _runtimeConfig.HostGlobalSettings.Mode is HostModeType.Development;
```

Comment:
The runtimeConfigProvider already has this implemented. Can we please invoke that? Trying to keep single implementation